Spigot-Server-Patches/0698-forced-whitelist-use-configurable-kick-message.patch

```diff
@@ -0,0 +1,19 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Trigary <trigary0@gmail.com>
+Date: Sat, 27 Mar 2021 09:24:23 +0100
+Subject: [PATCH] forced whitelist: use configurable kick message
+
+
+diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
+index c372e6897c73fad6f9c478cec9f58303fb0cce17..77f8be879ba098aaa15a5fd947dc3a4b39c5e397 100644
+--- a/src/main/java/net/minecraft/server/MinecraftServer.java
++++ b/src/main/java/net/minecraft/server/MinecraftServer.java
+@@ -2030,7 +2030,7 @@ public abstract class MinecraftServer extends IAsyncTaskHandlerReentrant<TickTas
+                 EntityPlayer entityplayer = (EntityPlayer) iterator.next();
+ 
+                 if (!whitelist.isWhitelisted(entityplayer.getProfile())) {
+-                    entityplayer.playerConnection.disconnect(new ChatMessage("multiplayer.disconnect.not_whitelisted"));
```

Comment:
Sounds like someone at Mojang forgot to create a translatable component.
Without the need of a custom message I would have preferred replacing it with a translatable component.
Due the need of a non plugin custom message, it's fine how it is handled in this pr.

Reply:
```json
  "multiplayer.disconnect.not_whitelisted": "You are not white-listed on this server!",
```

seems to exist in en_us.json

Reply:
That is a translatable component, just how mojang works is weird, according to the issue that all works as intended, just, they wish the whitelist message feature carried over